edb/schema/functions.py

```diff
@@ -740,6 +740,9 @@ class CallableObject(
     abstract = so.SchemaField(
         bool, default=False, inheritable=False, compcoef=0.909)
 
+    impl_is_strict = so.SchemaField(
+        bool, default=True, compcoef=0.4)
```

Comment:
We have to enforce this by 1) Creating functions with `STRICT` if this is `True`; or 2) For `FROM SQL FUNCTION` functions, validate that the underlying function is `STRICT`.

Reply:
1. Currently we just ignore this on user created functions, though I'll make it go and error instead if it is explicitly specified.

2. Do you know of a way to automatically do that validation?

Reply:
See how we use `sql_rval_consistency_check`.  This check would be a lot simpler, you'd only need to query `pg_proc.proisstrict` and `edgedb.raise` if it doesn not match `impl_is_strict`.

Reply:
> if it is explicitly specified

It should be impossible to specify in user DDL unless the Field is declared with `allow_ddl_set`, so we're good there.